PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -112,6 +136,20 @@ log_Check_Env_Var FSLDIR
 HCPPIPEDIR_PostFS="$HCPPIPEDIR/PostFreeSurfer/scripts"
 PipelineScripts="$HCPPIPEDIR_PostFS"
 
+#ExperimentRoot points to actual experiment directory
+ExperimentRoot=$Subject
+IsLongitudinal=0
+if [ "$LongitudinalMode" == "TIMEPOINT_STAGE1" -o "$LongitudinalMode" == "TIMEPOINT_STAGE2" ]; then
+    ExperimentRoot="$LongitudinalTimepoint".long."$LongitudinalTemplate"
+    IsLongitudinal=1
+elif [ "$LongitudinalMode" == "TEMPLATE" ]; then
+    ExperimentRoot="$Subject".long."$LongitudinalTemplate"
+    IsLongitudinal=1
+fi
```

Comment:
I don't see sanity checking for typos on this value.  Please do something like this for any "must be one of these possible values" argument:

```suggestion
case "$LongitudinalMode" in
    (NONE)
        ExperimentRoot=$Subject
        IsLongitudinal=0
        ;;
    (TIMEPOINT_STAGE1|TIMEPOINT_STAGE2)
        ExperimentRoot="$LongitudinalTimepoint".long."$LongitudinalTemplate"
        IsLongitudinal=1
        ;;
    (TEMPLATE)
        ExperimentRoot="$Subject".long."$LongitudinalTemplate"
        IsLongitudinal=1
        ;;
    (*)
        log_Err_Abort "unrecognized value for --longitudinal_mode: $LongitudinalMode"
        ;;
esac
```

Reply:
I added a check for typos in LongitudinalMode input value.